ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageVideoPlayer.swift

```diff
@@ -242,7 +261,7 @@ class NewTabPageVideoPlayer {
     playerObserver?.invalidate()
   }
 
-  private func parseStopFrameFromFilename(filename: String) -> Int? {
+  private func parseStopFrameNumberFromFilename(filename: String) -> Int? {
```

Comment:
nit: `parseStopFrameFromFilename` as not sure we need `Number`

Reply:
Fixed